app/models/Departure.scala

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
This is also going to be a problem with arrivals as I need to include the necessary code. It may be better doing a GET after all rather than calling the DB directly.

Reply:
Calling the DB directly is going to be useful if we can use it to expose correlationIds. Using the standard GET however won't expose failed messages and other things that are useful for a test scenario.

I'm not suggesting removing it, but if we can reduce the maintenance load, why not?